tests/pytests/unit/utils/test_crypt.py

```diff
@@ -1,11 +1,16 @@
 """
 Unit tests for salt.utils.crypt.py
 """
-
+import pytest
 
 import salt.utils.crypt
 from tests.support.mock import patch
 
+pytestmark = [
+    pytest.mark.windows_whitelisted,
```

Comment:
Aren't unit tests white listed by default?

Reply:
I didn't know that...